scripts/main.js

```diff
@@ -124,8 +131,8 @@ var listingImageDisplay = function(pin) {
     return listingImage;
 };
 
-var addMarkersToMap = function (infoWindow) {
-    pins.forEach( function (pin) {
+var addMarkersToMap = function (infoWindow, visiblePins) {
+    visiblePins.forEach( function (pin) {
         var markerIcon = {
```

Comment:
Is visiblePins the same as the var visibleMarkers declared on line 61 and thereafter, used in the getVisibleMarkers() and showVisibleMarkers() functions?

Reply:
visiblePins gets passed the value of visibleMarkers.  I didn't change the name within the addMarkersToMap function because I was already using the term 'marker' within that function